docs/reference/changelog-r2021.md

```diff
@@ -4,7 +4,7 @@
 Released on XX, XXth, 2021.
 
 - Bug fixes:
-  - Fixed calculation of front_speed_sum so that both front wheels are considered ([#3546](https://github.com/cyberbotics/webots/pull/3546))
+  - Fixed calculation of `front_speed_sum`in the Driver library so that both front wheels are considered in case of 4x4 cars ([#3546](https://github.com/cyberbotics/webots/pull/3546)).
```

Comment:
```suggestion
  - Fixed calculation of `front_speed_sum` in the Driver library so that both front wheels are considered in case of 4x4 cars ([#3546](https://github.com/cyberbotics/webots/pull/3546)).
```